app/models/game.rb

```diff
@@ -43,4 +37,18 @@ def winning_team
     end
     team
   end
+
+  def maximum_score
+    MAX_POINTS_PER_GAME
```

Comment:
It just seemed weird to have a method return `8`... It also seems weird to have an instance return a constant, but maybe that's just me. I _do_ think we should still consider making this configurable at the app level though, along with all that other stuff.